@xen-orchestra/lite/src/stories/web-core/button/ui-button.story.vue

```diff
@@ -2,13 +2,13 @@
   <ComponentStory
     v-slot="{ properties, settings }"
     :params="[
-      prop('color')
+      prop('accent')
         .type('ButtonColor')
```

Comment:
The type can be renamed to match the real type in the button component:
```suggestion
        .type('ButtonAccent')
```